### PR TITLE
fix(ClusterMetricsCores): format value and capacity same way

### PR DESCRIPTION
**Pull request by Raubzeug**
[Stand](https://nda.ya.ru/t/91u9DdHy79AMLo)

### Test Status: <span style="color: green;">✅ PASSED</span>
📊 [Full Report](https://ydb-platform.github.io/ydb-embedded-ui/1532/)

| Total | Passed | Failed | Flaky | Skipped |
|:-----:|:------:|:------:|:-----:|:-------:|
| 134 | 134 | 0 | 0 | 0 |

### Bundle Size: 🔺
Current: 79.11 MB | Main: 79.10 MB
Diff: +0.01 MB (0.01%)

⚠️ Bundle size increased. Please review.

<details>
<summary>ℹ️ CI Information</summary>

- Test recordings for failed tests are available in the full report.
- Bundle size is measured for the entire 'dist' directory.
- 📊 indicates links to detailed reports.
- 🔺 indicates increase, 🔽 decrease, and ✅ no change in bundle size.
</details>

### CODE REVIEW

File: src/utils/bytesParsers/formatNumber.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+export type Digits = keyof typeof sizes;
+
+/**
+ * This function is needed to keep more than 3 digits of the same size.
+ *
+ * @param significantDigits - number of digits above 3
+ * @returns size to format value to get required number of digits
+ *
+ * By default value converted to the next size when it's above 1000,
+ * so we have 900mb and 1gb. To extend it additional significantDigits could be set
```
Comment (astandrik) [inline src/utils/bytesParsers/formatNumber.ts:35]: I suppose mb and gb are not accurate examples as gb is 1024mb

File: src/utils/bytesParsers/i18n/ru.json [hunk 2]
```diff
@@ -1,8 +0,0 @@
-{
```
Comment (astandrik) [inline src/utils/bytesParsers/i18n/ru.json:1]: why?

File: src/utils/bytesParsers/formatNumber.ts [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import {formatNumber, roundToPrecision} from '../dataFormatters/dataFormatters';
```
Comment (astandrik) [inline src/utils/bytesParsers/formatNumber.ts:1]: I suppose formatNumber shouldn't be in bytesParsers as it has small relevance to bytes operations logics

File: src/utils/dataFormatters/__test__/formatNumbers.test.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        const result = formatNumericValues(undefined, 1_000_000);
+        expect(result).toEqual(['', `1${UNBREAKABLE_GAP}m`]);
+    });
+
+    it('should format both value and total correctly', () => {
+        const result = formatNumericValues(1024, 2048);
+        expect(result).toEqual(['1', `2${UNBREAKABLE_GAP}k`]);
+    });
+    it('should format value with label if set', () => {
+        const result = formatNumericValues(1024, 2048, undefined, undefined, true);
```
[the remark this answers is not in the document]
Comment (astandrik) [reply src/utils/dataFormatters/__test__/formatNumbers.test.ts:25]: maybe passing objects with comprehensible fielhds could help

File: src/utils/bytesParsers/formatNumber.ts [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+ * significantDigits = 1 - 9 000k and 10m
+ *
+ * significantDigits = 2 - 90 000m and 100b
+ *
+ * significantDigits = 3 - 900 000b and 1000t
+ */
+export const getNumberWithSignificantDigits = (value: number, significantDigits: number) => {
+    const multiplier = 10 ** significantDigits;
+
+    const thousandLevel = sizes.million.value * multiplier;
```
Comment (astandrik) [inline src/utils/bytesParsers/formatNumber.ts:48]: thousand.value

File: src/utils/bytesParsers/formatNumber.ts (hunk 1 above)
Comment (Raubzeug) [reply src/utils/bytesParsers/formatNumber.ts:35]: oh this terrible copy-paste -_-

File: src/utils/bytesParsers/i18n/ru.json (hunk 2 above)
Comment (Raubzeug) [reply src/utils/bytesParsers/i18n/ru.json:1]: Time ago decided not to support /ru localization and if change en keysets, remove ru version

File: src/utils/dataFormatters/__test__/formatNumbers.test.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (Raubzeug) [reply src/utils/dataFormatters/__test__/formatNumbers.test.ts:25]: I wanted to make this function with similar api to `formatBytes`. If you put a cursor into args, ide will suggest you comprehensive args names, you think it's not enough?

[the remark this answers is not in the document]
Comment (astandrik) [reply src/utils/dataFormatters/__test__/formatNumbers.test.ts:25]: I think this is impossible to decipher =)

```
const result = formatNumericValues(5120, 10240, 'billion', '/');
expect(result).toEqual(['0.00001', '0/b']);
```

File: src/utils/bytesParsers/i18n/ru.json (hunk 2 above)
Comment (astandrik) [reply src/utils/bytesParsers/i18n/ru.json:1]: As far as I remember we decided to support ru localization

File: src/utils/dataFormatters/__test__/formatNumbers.test.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (astandrik) [reply src/utils/dataFormatters/__test__/formatNumbers.test.ts:25]: formatBytes has object as and argument

File: src/utils/dataFormatters/formatNumber.ts [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+    const thousandLevel = sizes.thousand.value * multiplier;
+    const millionLevel = sizes.million.value * multiplier;
+    const billionLevel = sizes.billion.value * multiplier;
+    const trillionLevel = sizes.trillion.value * multiplier;
+
+    let size: Digits = 'thousand';
+
+    if (value > thousandLevel) {
+        size = 'thousand';
```
[the remark this answers is not in the document]
Comment (astandrik) [reply src/utils/dataFormatters/formatNumber.ts:56]: Oh, I see, its key of sized dict

**Review by astandrik — APPROVED** [review]

**Pull request merged**